lib/inspec/fetcher/git.rb

```diff
@@ -1,7 +1,7 @@
-require "tmpdir" unless defined?(Dir.mktmpdir)
-require "fileutils" unless defined?(FileUtils)
-require "mixlib/shellout" unless defined?(Mixlib::ShellOut)
-require "inspec/log"
+require 'tmpdir' unless defined?(Dir.mktmpdir)
+require 'fileutils' unless defined?(FileUtils)
+require 'mixlib/shellout' unless defined?(Mixlib::ShellOut)
+require 'inspec/log'
```

Comment:
Keep the double quotes that is convention that Chef Inspec follows as per my understanding. This change will fail you lint test.

Reply:
To run the linter, run `bundle exec chefstyle` and to autocorrect, run `bundle exec chefstyle -a`

Reply:
Got it! I will make the changes as per the chef style.